startup_scripts/000_users.py

```diff
@@ -13,6 +13,12 @@
         user = User.objects.create_user(
             username=username,
             password=user_details.get("password", 0) or User.objects.make_random_password(),
+            first_name=user_details.get("first_name", "no-name"),
+            last_name=user_details.get("last_name", "no-lastname"),
+            email=user_details.get("email", "none"),
+            is_staff=user_details.get("is_staff", False),
+            is_active=user_details.get("is_active", False),
```

Comment:
If `is_active` is to default to False (I don't think it should), then the examples [here](https://github.com/netbox-community/netbox-docker/blob/release/initializers/users.yml) should probably mention that. Not sure preloading a disabled user is common.